ui/src/preview/preview-confirm-revert-changes-dialog.tsx

```diff
@@ -153,7 +153,8 @@ export const PreviewConfirmRevertChangesDialog: React.FC<PreviewRejectConfirmDia
         <Dialog
             title={t('publish.revert-confirm.title')}
             variant={DialogVariant.LIGHT}
-            allowClose={false}
+            allowClose={!isReverting}
```

Comment:
sulkemisnappia ei voi disabloida helposti ulkoapäin, joten wiretin sen vain koko ruksin näkyvyyteen. Eli se katoaa siksi aikaa kun hylkääminen on käynnissä, minkä jälkeen koko dialogi sulkeutuu muutenkin